todos.md

```diff
@@ -1,6 +1,4 @@
-- Build minimal Nixos AMI
```

Comment:
I still want this

Reply:
Good point. A custom built AMI with no root login would also mean we *could* even forgo having a public facing IP address (which also saves some cost).